components/MermaidDiagram.module.css

```
@@ -18,3 +18,46 @@
+.mermaidTarget :global(foreignObject > div),
+.mermaidTarget :global(.nodeLabel),
+.mermaidTarget :global(.edgeLabel) {
+    overflow: visible;
+}
+
+/*
+ * ノードラベルの文字色（ノードラベル限定）。
+ * 背景がソリッド濃色のノードは明るい白で統一し、<br/> 2 行目が暗く残る問題も解消する。
+ * エッジラベル / シーケンス図 Note は対象外（テーマの暗背景＋明文字のまま）。
+ */
+.mermaidTarget :global(.node .nodeLabel),
```

Biome (2.4.16)

[error] Unexpected unknown pseudo-class global 

(lint/correctness/noUnknownPseudoClass)